### ISSUE TITLE
IE Wooden Storage Crate contents lost when broken

### ISSUE THREAD

**Issue by bluelightning32** [SpongePowered/SpongeForge#2709]
**I am currently running**
- SpongeForge version: spongeforge-1.12.2-2768-7.1.6-RC3671.jar
- Forge version: forge-1.12.2-14.23.5.2822-universal.jar
- Java version: 1.8.0_201
- Operating System: Fedora 29 64bit

- Plugins/Mods:
```
[20:41:45] [Server thread/INFO] [minecraft/DedicatedServer]: Plugins (9): Minecraft, Minecraft Coder Pack, SpongeAPI, Sponge, Inventory Sync, JdaLibraryLoader,
LuckPerms, UltimateChat, spark
```

- Modpack: DD&SS 4.10a, which includes ImmersiveEngineering-0.12-89.jar

**Issue Description**
Repro steps:
1. Place down an Immersive Engineering Wooden Storage Crate.
2. Put something in the crate.
3. Break the crate.
4. With Auto Pickup (part of modpack), the crate goes directly into your inventory.
5. Place crate back down. It should keep the inventory, but instead all contents are lost.

The problem also occurs with the IE reinforced storage crate.

This bug does not reproduce in single player. This is probably related to https://github.com/SpongePowered/SpongeForge/issues/2700.

**Comment by phit** [comment]
please retest with 7.1.6-RC3683

**Comment by bluelightning32** [comment]
I retested with spongeforge-1.12.2-2768-7.1.6-RC3684.jar. It reproduces in exactly the same way.

**Comment by bluelightning32** [comment]
Still reproduces with spongeforge-1.12.2-2825-7.1.6-RC3692.jar.

**Comment by bluelightning32** [comment]
Confirmed fixed in spongeforge-1.12.2-2825-7.1.6-RC3693.jar. Thanks!
